src/common/Pagination/styled.js

```diff
@@ -22,7 +22,8 @@ export const Button = styled.button`
   display: flex;
   align-items: center;
   gap: 6px;
-  transition: 1s;
+  transition: 0.5ms;
```

Comment:
It might be too fast to see the transition now 😅 
Please, consider using seconds instead of milliseconds 👌

Reply:
oops.. typo error. Corrected